neighbour-model/src/main/java/no/vegvesen/ixn/federation/repository/PrivateChannelRepository.java

```diff
@@ -0,0 +1,18 @@
+package no.vegvesen.ixn.federation.repository;
+
+import no.vegvesen.ixn.federation.model.PrivateChannel;
+import no.vegvesen.ixn.federation.model.PrivateChannelStatus;
+import org.springframework.data.repository.CrudRepository;
+import java.util.List;
+
+public interface PrivateChannelRepository extends CrudRepository<PrivateChannel, Integer> {
+    List<PrivateChannel> findAllByServiceProviderName(String serviceProviderName);
+    List<PrivateChannel> findAllByStatusAndServiceProviderName(PrivateChannelStatus privateChannelStatus, String serviceProviderName);
+    PrivateChannel findByServiceProviderNameAndIdAndStatusIsNot(String serviceProviderName, Integer Id, PrivateChannelStatus status);
+    PrivateChannel findByServiceProviderNameAndId(String serviceProviderName, Integer id);
+
+    List<PrivateChannel> findAll();
```

Comment:
No need to override findAll, it's already there in the repo interface, and annotated.